### PR TITLE
Fix non-critical `getRangeAt(0)` errors

### PR DESCRIPTION
**Pull request by Soreine**
This should fix non-fatal errors: 
`IndexSizeError: Failed to execute 'getRangeAt' on 'Selection': 0 is not a valid index`  
https://sentry.io/organizations/gitbook/issues/737882435

### CODE REVIEW

**Comment by Soreine** [comment]
Yep, I'll report the error, without throwing to keep the rest of the logic running.
I need to dig what can cause selections without any ranges

**Comment by Soreine** [comment]
> Before the user has clicked a freshly loaded page, the rangeCount is 0. After the user clicks on the page, rangeCount is 1, even if no selection is visible.
> 
> A user can normally only select one range at a time, so the rangeCount will usually be 1. Scripting can be used to make the selection contain more than one range.
https://developer.mozilla.org/en-US/docs/Web/API/Selection/rangeCount

There's no apparent reason for a selection to have 0 range except before the user click on the page. For `scrollToSelection`, this can logically happen if the document updates before the user click in the document. So I'm ignoring the error there.
However `copyFragment` should only be called with a selection, so I report an error in this case.

**Comment by Soreine** [comment]
At least, with a normal error reporting, we will have proper stacktraces on sentry

File: packages/slate-debug/src/index.ts [hunk 1]
```diff
@@ -59,10 +50,21 @@ function deprecate(version: string, message: string, ...args: any[]) {
...
     deprecate,
-    error,
-    warn
+    warn,
+    report,
+    Logger: Debug
 };
 
-export default logger;
+export default debug;
```
Comment (SamyPesse) [inline packages/slate-debug/src/index.ts]: It's a good occasion de stop using default exports and instead import as ESM module:

```js
import { Logger } from '@gitbook/slate-debug';
```

**Pull request merged**